rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -93,13 +93,11 @@ void main_window::Init()
 	Q_EMIT RequestGlobalStylesheetChange(m_gui_settings->GetCurrentStylesheetPath());
 	ConfigureGuiFromSettings(true);
 
-#ifdef BRANCH
-	if ("RPCS3/rpcs3/master"s != STRINGIZE(BRANCH) && ""s != STRINGIZE(BRANCH))
-#else
-	if (false)
-#endif
+	std::string_view branch_name = rpcs3::get_full_branch();
+
+	if (bool is_master = branch_name == "RPCS3/rpcs3/master"s; !is_master)
 	{
-		gui_log.warning("Experimental Build Warning! Build origin: " STRINGIZE(BRANCH));
+		gui_log.warning("Experimental Build Warning! Build origin: ", branch_name);
```

Comment:
Since the branch is now an argument instead of combined string literal, I think you need an %s to mark where it should be placed.

Reply:
You caught my 5am brain at work :smile:. Nice catch.